plugins/modded_entity_fabric.js

```diff
@@ -81,14 +81,13 @@ function setTemplate() {
 		renderer: `%(bone).render(f5);`,
 		cube: `%(bone).boxes.add(new Box(%(bone), %(uv_x), %(uv_y), %(x), %(y), %(z), %(dx), %(dy), %(dz), %(inflate), %(mirror)));`,
 	};
-
-	Codecs.modded_entity.templates['Fabric 1.15+'] = {
-		name: 'Fabric 1.15+',
+	Codecs.modded_entity.templates['Fabric 1.15-1.16'] = {

```

Comment:
Yes, it would fall back to the latest Forge version if the version that the project was saved in no longer exists.
I would recommend instead to only change the name, not the ID.